warehouse/models/mart/benefits/fct_benefits_events.sql

```diff
@@ -1,7 +1,65 @@
 {{ config(materialized='table') }}
 
 WITH fct_benefits_events AS (
-    SELECT * FROM {{ ref('stg_amplitude__benefits_events') }}
+    SELECT
+        -- Only fields that aren't _always_ empty (https://dashboards.calitp.org/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjM1ODR9LCJ0eXBlIjoicXVlcnkifSwiZGlzcGxheSI6InRhYmxlIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319)
+        app,
+        device_id,
+        user_id,
+        client_event_time,
+        event_id,
+        session_id,
+        event_type,
+        version_name,
+        os_name,
+        os_version,
+        device_family,
+        device_type,
+        country,
+        language,
+        library,
+        city,
+        region,
+        event_properties,
+        user_properties,
```

Comment:
RE: @lauriemerrell's comment, I think we can drop these unparsed JSON columns from this table now that we have them parsed out. They won't be useful within Metabase.